knights/contract/player_control.hpp

```diff
@@ -193,6 +193,15 @@ class player_control : public control_base {
         return (int)(num + v1 + v2 + v3) % k;
     }
 
+    void require_sigle_action() {
+        char buffer[512];
+        int actual_size = read_transaction(buffer, 512);
+        eosio::datastream<const char *> ds(buffer, actual_size);
+        eosio::transaction tx;
+        ds >> tx;
+        eosio_assert((tx.actions.end() - tx.actions.begin()) == 2, "wrong number of actions in transaction");
```

Comment:
Hmm, I was thinking the number of actions should be 1.
It's 2 for wizards because they use transactions consisting of two actions (one `eosio.token` transfer and one `wizardstoken` command).